tests/JobCreatorTest.php

```diff
@@ -635,7 +623,7 @@ public function provideParentBranch(): array
                     parent_branch: '4.10-release'
                     EOT
                 ]),
-                'dev-4.10-release'
+                '4.11.x-dev'
```

Comment:
I've removed all of the logic for `-release` branches and removed most scenarios that tested against them - but I've kept this one (and just updated the expected value accordingly) so we do still have some check against what would happen with a branch with that suffix.